entrypoint-5.x.sh

```diff
@@ -73,6 +92,14 @@ else
   fi
 fi
 
+# check if properties file exists; create it if it is mising
```

Comment:
If it would be helpful, we could do something similar for the properties files as I am doing for the `data/html` directory which may be missing upon upgrades:
https://github.com/mbentley/docker-omada-controller/blob/master/install.sh#L191-L197

At install, that would allow it to be restored in a similar fashion:
https://github.com/mbentley/docker-omada-controller/blob/master/entrypoint-5.x.sh#L122-L129

I'd probably suggest that same approach to support that use case.  The main difference I think I would make would be that I would make it support untaring a single file to replace it so it would need a loop to check for both `log4j2.properties` and  `omada.properties` as I would not want to override someone's custom file if one exists and not both.

Reply:
True! I not thinked in that, i too would not want to override someone's custom file if one exists and not both

Reply:
If i understood correctly, now this problem is fixed with your suggest